.github/workflows/acpi-validation.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will run mock-acpi compose on GH runner and run validator tool
+
+name: Validate ACPI
+
+on: # yamllint disable-line rule:truthy
+  workflow_call:
+
+jobs:
+  validate-acpi:
+    name: Validate ACPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
```

Comment:
do we need specific python version?
if so, we may have to upgrade manually when GHA bump up runner OS version.

Reply:
Sure